packages/fast-development-site-react/src/components/shell/index.tsx

```diff
@@ -22,7 +22,7 @@ class Shell extends React.Component<{}, {}> {
     > = {
         container: {
             fontFamily: "Segoe UI, SegoeUI, Helvetica Neue, Helvetica, Arial, sans-serif",
-            fontSize: toPx(14),
+            fontSize: "11px",
```

Comment:
Why not use `toPx(11)` ?  Is there a difference?

Reply:
ToPX is more expensive. It’s runs a functions that converts a number to a string and appends “pix” to the end. It should only be used when you want to calculate some value. I think people just assumed they should use it because it was used everywhere. So really we should be ripping it out when we can.